### ISSUE TITLE
Can't do pip install libzt

### ISSUE THREAD

**Issue by GoldenMelky** [zerotier/libzt#192]
I can't do pip install libzt, when i do it it's shows:
`$ pip3 install libzt
Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not find a version that satisfies the requirement libzt (from versions: none)
ERROR: No matching distribution found for libzt`

**Comment by fregapple** [comment]
I am also having this issue on Windows with python 3.9.13

https://github.com/zerotier/libzt/issues/181#issuecomment-1214305148

**Comment by AJenbo** [comment]
the version on pypi appears to only be avalible for Linux/Mac Python 3.5 - 3.9

**Comment by fregapple** [comment]
Yeah I also noticed that. I did see in another issue, They said they were gonna make windows builds... This was august 2022. So perhaps they gave up on that?

**Comment by AJenbo** [comment]
The lack of Python 3.10+ support is also an issue, the documentation is full of flaws, there are lots of missing features and the 1.8 version of the extension doesn't work on some systems. All that made me eventually give up on this extension and instead write a C++ application that takes care of the ZT portion which the python script can then call.

**Comment by fregapple** [comment]
Would you be interested in sharing that application! Would be nice to use something better than python sockets!

**Comment by AJenbo** [comment]
its not general pourpos
